src/script/lua_api/l_client.cpp

```diff
@@ -144,13 +144,6 @@ int ModApiClient::l_show_formspec(lua_State *L)
 	return 1;
 }
 
-// send_respawn()
-int ModApiClient::l_send_respawn(lua_State *L)
```

Comment:
why this has been removed ? please keep it, you removed a CSM feature.

Reply:
Ok, i thought it wasn't an API, but it is.